.github/workflows/scheduled.yml

```diff
@@ -768,6 +768,87 @@ jobs:
             /tmp/server.log
 
 
+  presto-bias-java-single-function-expression-fuzzer-run:
```

Comment:
With this addition do we still need the non-java presto bias fuzzer run?

Reply:
Hi @assignUser, yes, we still need the non-java presto bias fuzzer job. This is because the added fuzzer job only run the single function added or modified by a PR, while the non-java bias fuzzer job runs other functions with the added or modified function as well and hence could provide some coverage not overlapped with the added fuzzer job.